src/vmtest.rs

```diff
@@ -119,6 +119,16 @@ impl Vmtest {
         target.kernel = target.kernel.map(|s| self.resolve_path(s.as_path()));
         target.rootfs = self.resolve_path(target.rootfs.as_path());
         target.vm.bios = target.vm.bios.map(|s| self.resolve_path(s.as_path()));
+        target.vm.mounts = target
+            .vm
+            .mounts
+            .iter()
+            .map(|(k, v)| {
+                let mut m = v.clone();
+                m.host_path = self.resolve_path(v.host_path.as_path());
+                (k.clone(), m)
```

Comment:
should be possible to modify in-place? Given we already have a `mut target` cloned above. don't worry about it if it's complicated - just a nit